xline/src/storage/kv_store.rs

```diff
@@ -45,7 +46,7 @@ pub(crate) struct KvStoreBackend {
     /// Key Index
     index: Index,
     /// DB to store key value
-    db: DB,
+    db: DB<Memory>,
```

Comment:
Refer to the comment in `AuthStoreBackend`

Reply:
fixed